subprojects/core/src/integTest/groovy/org/gradle/api/tasks/bundling/ZipIntegrationTest.groovy

```diff
@@ -27,7 +27,7 @@ import java.nio.charset.Charset
 @TestReproducibleArchives
 class ZipIntegrationTest extends AbstractIntegrationSpec {
 
-    def 'ensure duplicates not included by default'() {
+    def 'deprecation warning on duplicates with default strategy'() {
```

Comment:
I guess this should have a similar name in both test classes.

Reply:
Both tests are now in the same class with a `where` class.